tutorials/optimization/cpu_optimization.rst

```diff
@@ -186,73 +193,82 @@ Third parties provide support for several other languages, including `Rust
+pausing or hiding them. You don't have to delete a detached node. You can for
+example, keep a reference to a node, detach it from the scene tree using
+:ref:`Node.remove_child(node) <class_Node_method_remove_child>`, then reattach
+it later using :ref:`Node.add_child(node) <class_Node_method_add_child>`.
+This can be very useful for adding and removing areas from a game, for example.
 
 You can avoid the SceneTree altogether by using Server APIs. For more
 information, see :ref:`doc_using_servers`.
 
 Physics
 =======
 
-In some situations physics can end up becoming a bottleneck, particularly with
-complex worlds, and large numbers of physics objects.
+In some situations, physics can end up becoming a bottleneck. This is
+particularly the case with complex worlds and large numbers of physics objects.
 
-Some techniques to speed up physics:
+Here are some techniques to speed up physics:
 
-* Try using simplified versions of your rendered geometry for physics. Often
+- Try using simplified versions of your rendered geometry for collision shapes. Often,
   this won't be noticeable for end users, but can greatly increase performance.
-* Try removing objects from physics when they are out of view / outside the
+- Try removing objects from physics when they are out of view / outside the
   current area, or reusing physics objects (maybe you allow 8 monsters per area,
   for example, and reuse these).
 
-Another crucial aspect to physics is the physics tick rate. In some games you
+Another crucial aspect to physics is the physics tick rate. In some games, you
 can greatly reduce the tick rate, and instead of for example, updating physics
-60 times per second, you may update it at 20, or even 10 ticks per second. This
-can greatly reduce the CPU load.
+60 times per second, you may update them only 30 or even 20 times per second.
+This can greatly reduce the CPU load.
 
 The downside of changing physics tick rate is you can get jerky movement or
-jitter when the physics update rate does not match the frames rendered.
+jitter when the physics update rate does not match the frames per second
+rendered. Also, decreasing the physics tick rate will increase input lag.
+It's recommended to stick to the default physics tick rate (60 Hz) in most games
+that feature real-time player movement.
 
-The solution to this problem is 'fixed timestep interpolation', which involves
+The solution to jitter is to use *fixed timestep interpolation*, which involves
 smoothing the rendered positions and rotations over multiple frames to match the
-physics. You can either implement this yourself or use a third-party addon.
-Interpolation is a very cheap operation, performance wise, compared to running a
-physics tick, orders of magnitude faster, so this can be a significant win, as
-well as reducing jitter.
+physics. You can either implement this yourself or use a
+`third-party addon <https://github.com/lawnjelly/smoothing-addon>`__.
```

Comment:
I thought hyperlinks just had a single "_" at the end. I could be wrong though. If i am, please ignore.

Reply:
Both can be used, depending on your goals. A single underscore creates a named reference (you can't duplicate them without causing a Sphinx build warning). A double underscore creates an anonymous reference (you can duplicate them without causing a Sphinx build warning).

Due to this, I prefer using anonymous references unless they can't be used for some reason.